### ISSUE TITLE
[BUG?] Error Installing on Windows 10 64bit

### ISSUE THREAD

**Issue by angelhdzmultimedia** [arturadib/node-qt#48]
![image](https://user-images.githubusercontent.com/10191085/63991826-3d6eac80-cab7-11e9-8728-3abb5504a291.png)

**Comment by angelhdzmultimedia** [comment]
Hello?

**Comment by praisethemoon** [comment]
Looks like a python related issue, are you using python2?

**Comment by angelhdzmultimedia** [comment]
> Looks like a python related issue, are you using python2?

Thank you for replying.

Yes I noticed this uses Python 2 by the error outputs and I'm using python  3.7.4.

I hope this gets ported to Py3 soon.

**Comment by praisethemoon** [comment]
@angelhdz I dont think it will anytime soon :) always use virtual env especially on linux, good luck!

**Comment by angelhdzmultimedia** [comment]
@praisethemoon sorry I don't get it, are you suggesting me to use virtual env so I get it to work, or are you talking about this package using a virtual env and that's why it doesn't support Python 3?

**Comment by praisethemoon** [comment]
@angelhdz you use virtual env just to build your project, so if you install py2 it wont mess up your environment, just in case :) For example is my mac my default python is v3, whenever i build some native node modules, I Just switch to python. v2 on virtual env, then go back once build finishes. Note: Python foundation will go with python v3 as a long term version. Python v2 support will drop at some point.

**Comment by angelhdzmultimedia** [comment]
Thank you for replying. But I don't think that's good for me, because I don't want to code in Python 2. I started with Python 3 and I'm glad I did because in Python 2 many things were different link the print was a statement and now is a function.

So I'm not downgrading to Python 2  just to use this binding. I will keep using PyQt on Python then.

I just wanted to use Qt in Javascript/NodeJS because I prefer JS over Python.

I've been reading how I can make my own Qt binding in NodeJS but I haven't found anything.

Cheers!
